docs/content/ingestion/firehose.md

```diff
@@ -54,54 +94,74 @@ Sample spec:
 }
 ```
 
+|property|description|default|required?|
+|--------|-----------|-------|---------|
+|type|This should be "static-azure-blobstore".|N/A|yes|
+|blobs|JSON array of [Azure blobs](https://msdn.microsoft.com/en-us/library/azure/ee691964.aspx).|N/A|yes|
+
+Azure Blobs:
+
+|property|description|default|required?|
+|--------|-----------|-------|---------|
+|container|Name of the azure [container](https://azure.microsoft.com/en-us/documentation/articles/storage-dotnet-how-to-use-blobs/#create-a-container)|N/A|yes|
+|path|The path where data is located.|N/A|yes|
+
 #### TwitterSpritzerFirehose
 
-See [Examples](../tutorials/examples.html). This firehose connects directly to the twitter spritzer data stream.
+This firehose connects directly to the twitter spritzer data stream.
+
+Sample spec:
 
-#### RandomFirehose
+```json
+"firehose" : {
+    "type" : "twitzer",
+    "maxEventCount": -1,
+    "maxRunMinutes": 0
+}
+```
 
-See [Examples](../tutorials/examples.html). This firehose creates a stream of random numbers.
+|property|description|default|required?|
+|--------|-----------|-------|---------|
+|type|This should be "twitzer"|N/A|yes|
+|maxEventCount|max events to receive, -1 is infinite, 0 means nothing is delivered; use this to prevent infinite space consumption or to prevent getting throttled at an inconvenient time.|N/A|yes|
+|maxRunMinutes|maximum number of minutes to fetch Twitter events.  Use this to prevent getting throttled at an inconvenient time. If zero or less, no time limit for run.|N/A|yes|
 
-#### RabbitMqFirehose
+#### RabbitMQFirehose
 
 This firehose ingests events from a define rabbit-mq queue.
-<br>
+
 **Note:** Add **amqp-client-3.2.1.jar** to lib directory of druid to use this firehose.
```

Comment:
I'm not sure, I've never used the rabbitmq firehose, I'd rather just keep whta ppl have been successful with